test/functional/helpers/assertions/responseStatus.js

```diff
@@ -1,19 +1,17 @@
 import { t } from "testcafe";
 
-const responseStatus = (networkLogs, status) => {
-  return Promise.all(
-    networkLogs.map(req => {
-      // TODO: Check why some requests don't have responses.
-      // TODO: It looks like test cafe might not be handling gzip correctly.
-      if (!req.response) {
-        return Promise.resolve();
-      }
-
-      return t
+const responseStatus = async (networkLogs, status) => {
+  for (let i = 0; i < networkLogs.length; i += 1) {
+    const req = networkLogs[i];
+    // TODO: Check why some requests don't have responses.
+    // TODO: It looks like test cafe might not be handling gzip correctly.
+    if (!req.response) {
+      // eslint-disable-next-line no-await-in-loop
+      await t
         .expect(req.response.statusCode === status)
         .ok(`expected ${status} to be found in ${req.response.statusCode}`);
-    })
-  );
+    }
+  }
```

Comment:
TestCafe was complaining about this. We're supposed to await each testcafe call individually so that order is maintained.